.github/workflows/gh-pages.yaml

```diff
@@ -0,0 +1,53 @@
+name: Storybook GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'WordPress/openverse-frontend' }}
+
+    steps:
+      - name: checkout
```

Comment:
I'm being shamefully obsessive here but all workflows use start-case in step names. Eg:
- Checkout
- Setup Node.js environment